magpie/adapter/magpieprocess.py

```diff
@@ -403,7 +403,7 @@ def is_visible_by_user(self, ems_processes_id, process_id, request):
             return False
 
         # use inherited flag to consider both user and group permissions on the resource
-        path = '{host}/users/{usr}/resources/{res}/permissions?inherit=true' \
+        path = '{host}/users/{usr}/resources/{res}/permissions?inherit=true&effective=true' \
```

Comment:
Maybe a comment also on the effective flag...

Reply:
done in below commits